ports/raspberrypi/boards/sparkfun_thing_plus_rp2350/pins.c

```diff
@@ -0,0 +1,63 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2024 Scott Shawcroft for Adafruit Industries
+//
+// SPDX-License-Identifier: MIT
+
+#include "shared-bindings/board/__init__.h"
+
+static const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
+
+    { MP_ROM_QSTR(MP_QSTR_D0), MP_ROM_PTR(&pin_GPIO0) },
+    { MP_ROM_QSTR(MP_QSTR_D1), MP_ROM_PTR(&pin_GPIO1) },
+    { MP_ROM_QSTR(MP_QSTR_D2), MP_ROM_PTR(&pin_GPIO2) },
+    { MP_ROM_QSTR(MP_QSTR_D3), MP_ROM_PTR(&pin_GPIO3) },
+    { MP_ROM_QSTR(MP_QSTR_D4), MP_ROM_PTR(&pin_GPIO4) },
+    { MP_ROM_QSTR(MP_QSTR_D5), MP_ROM_PTR(&pin_GPIO5) },
```

Comment:
I don't have the Thing Plus RP2350 to check, but I compared to the Thing Plus RP2040 and saw there's no "D5" pin broken out.

In general, if a pin is not used, the corresponding entry in the board pins file will be omitted.

Reply:
There's some minor pinout differences between the boards, and GPIO 5 is now broken out. I've double checked that the pins in this file match the pins available to the user. The only pins that are not available are GPIO 8 for the PSRAM, and GPIO 23/24/25/29 for the radio.